Cargo.toml

```diff
@@ -104,108 +108,119 @@ impl-trait-for-tuples = { git = "https://github.com/sora-xor/impl-trait-for-tupl
+sp-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+sp-keyring = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+sp-mmr-primitives = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+sp-maybe-compressed-blob = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+substrate-frame-rpc-system = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+mmr-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+pallet-mmr = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+substrate-prometheus-endpoint = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+sp-beefy = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+pallet-beefy = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+pallet-beefy-mmr = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+beefy-gadget = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+beefy-gadget-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+beefy-merkle-tree = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+pallet-preimage = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+substrate-wasm-builder = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+try-runtime-cli = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
+
+pallet-staking = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
 
-pallet-staking = { git = "https://github.com/soramitsu/sora2-frame-pallets.git", branch = "polkadot-v0.9.25-lock-fix" }
+# [patch."https://github.com/sora-xor/sora2-common.git"]
+# beefy-light-client = { path = "../sora2-common/pallets/beefy-light-client" }
+# beefy-light-client-rpc = { path = "../sora2-common/pallets/beefy-light-client/rpc" }
+# beefy-light-client-runtime-api = { path = "../sora2-common/pallets/beefy-light-client/runtime-api" }
+# leaf-provider = { path = "../sora2-common/pallets/leaf-provider" }
+# leaf-provider-rpc = { path = "../sora2-common/pallets/leaf-provider/rpc" }
+# leaf-provider-runtime-api = { path = "../sora2-common/pallets/leaf-provider/runtime-api" }
+# bridge-common = { path = "../sora2-common/pallets/bridge-common" }
+# dispatch = { path = "../sora2-common/pallets/dispatch" }
+# substrate-bridge-app = { path = "../sora2-common/pallets/substrate-app" }
+# substrate-bridge-channel = { path = "../sora2-common/pallets/substrate-channel" }
+# substrate-bridge-channel-rpc = { path = "../sora2-common/pallets/substrate-channel/rpc" }
+# bridge-types = { path = "../sora2-common/pallets/types" }
```

Comment:
Should we remove it?

Reply:
It useful for testing when you change something in sora2-common repo, so I think we can keep it